roborock/code_mappings.py

```diff
@@ -214,7 +215,7 @@ class RoborockMopIntensityV2(RoborockMopIntensityCode):
     low = 201
     medium = 202
     high = 203
-    custom = 207
+    custom = 204
```

Comment:
Can you make a new enum like we do above for the s7 but for your device instead of modifying this one

Reply:
Now, it requires a much more proper testing locally, I will do tonight :)